pkg/webhook/policy.go

```diff
@@ -157,6 +167,16 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 		}
 	}()
 
+	// namespace is excluded from webhook using config match
+	if h.configMatchSet != nil && len(h.configMatchSet.ExcludedNamespaces[match.Webhook]) > 0 {
+		for _, ns := range h.configMatchSet.ExcludedNamespaces[match.Webhook] {
+			if ns == req.AdmissionRequest.Namespace {
+				requestResponse = allowResponse
+				return admission.ValidationResponse(true, "")
```

Comment:
Is it possible to put as a justification that the namespace is being ignored for admission?

Reply:
added justification response